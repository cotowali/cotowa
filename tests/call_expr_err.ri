fn f() {
  unknown_fn()
}
