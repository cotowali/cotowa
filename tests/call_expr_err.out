call_expr_err.ri 5,2: function `unknown_fn` is not defined
  > unknown_fn()
    ^^^^^^^^^^^^
call_expr_err.ri 8,2: `v` is not function (`int`)
  > v()
    ^^^
call_expr_err.ri 11,4: mismatched types: `int` (expected) and `string` (found)
  > "a"
    ^^^
call_expr_err.ri 12,2: expected 1 arguments, but got 0
  > f()
    ^^^
call_expr_err.ri 13,2: expected 1 arguments, but got 2
  > f(0, 1)
    ^^^^^^^
